week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -85,59 +96,71 @@ const alphaStudentGroup = [
   ["Tayoa", 11],
   ["Nina", 10],
   ["Bob", 9],
-  ["Lee", 1]
-]
+  ["Lee", 1],
+];
 
 const deltaStudentGroup = [
   ["Zoidber", 6],
   ["Bender", 5],
   ["Zapp", 7],
-  ["amy", 0]
-]
+  ["amy", 0],
+];
 
 function arraysEqual(a, b) {
-    if (a === b) return true;
-    if (a == null || b == null) return false;
-    if (a.length != b.length) return false;
-  
-    for (let i = 0; i < a.length; ++i) {
-      if (a[i] !== b[i]) return false;
-    }
-  
-    return true;
+  if (a === b) return true;
+  if (a == null || b == null) return false;
+  if (a.length != b.length) return false;
+
+  for (let i = 0; i < a.length; ++i) {
+    if (a[i] !== b[i]) return false;
+  }
+
+  return true;
 }
 
 function test(test_name, expr) {
-    let status;
-    if (expr) {
-      status = "PASSED";
-    } else {
-      status = "FAILED";
-    }
-  
-    console.log(`${test_name}: ${status}`);
+  let status;
+  if (expr) {
+    status = "PASSED";
+  } else {
+    status = "FAILED";
+  }
+
+  console.log(`${test_name}: ${status}`);
 }
 
-test("eligibleStudents function works",
-  arraysEqual(getEligibleStudents(alphaStudentGroup), ["Ahmed", "Clement", "Tayoa", "Nina", "Bob"])
-)
-
-test("eligibleStudents function without eligible candidates",
-  arraysEqual(getEligibleStudents(deltaStudentGroup), [])
-)
-
-test("eligibleStudents function with no candidates",
-  arraysEqual(getEligibleStudents([]), [])
-)
+test(
+  "eligibleStudents function works",
+  arraysEqual(getEligibleStudents2(alphaStudentGroup), [
+    "Ahmed",
+    "Clement",
+    "Tayoa",
+    "Nina",
+    "Bob",
+  ])
+);
+
+test(
+  "eligibleStudents function without eligible candidates",
+  arraysEqual(getEligibleStudents2(deltaStudentGroup), [])
+);
+
+test(
+  "eligibleStudents function with no candidates",
+  arraysEqual(getEligibleStudents2([]), [])
+);
 
```

Comment:
Hmm ...

Don't recommend you messing about around with the test.

Just to let you know both functions

```diff
+ getEligibleStudents()
- getEligibleStudents2()
```

Are not the same function, therefore they have separate tests, please change these back to the original and do the exercises for `getEligibleStudents()`, not `getEligibleStudents2()`. Again please